#@module_name hsr_pick_up_object
string object_id
---
bool success
